ush/setup.py

```diff
@@ -527,81 +527,32 @@ def get_location(xcs,fmt):
     # -----------------------------------------------------------------------
     #
     # Check that DATE_FIRST_CYCL and DATE_LAST_CYCL are strings consisting
-    # of exactly 8 digits.
+    # of exactly 10 digits.
     #
     # -----------------------------------------------------------------------
     #
     if not isinstance(DATE_FIRST_CYCL, datetime.date):
         print_err_msg_exit(
             f'''
             DATE_FIRST_CYCL must be a string consisting of exactly 8 digits of the
-            form \"YYYYMMDD\", where YYYY is the 4-digit year, MM is the 2-digit
-            month, and DD is the 2-digit day-of-month.
+            form \"YYYYMMDDHH\", where YYYY is the 4-digit year, MM is the 2-digit
+            month, DD is the 2-digit day-of-month, and HH is the 2-digit
+            cycle hour.
               DATE_FIRST_CYCL = \"{DATE_FIRST_CYCL}\"'''
         )
```

Comment:
@danielabdi-noaa If there was some check on these being 8 digits, I couldn't find it. I saw that it was built-in to use however many digits provided.

Reply:
@christinaholtNOAA Yes, it should handle it.